lib/repo_tools.pm

```diff
@@ -267,7 +267,7 @@ sub rmt_wizard {
 
     enter_cmd "yast2 rmt;echo yast2-rmt-wizard-\$? > /dev/$serialdev";
     # On x11, workaround bsc#1191112 by mouse click or drag the dialog.
-    if (($setup_console =~ /x11/) && (check_var('RMT_TEST', 'rmt_chinese'))) {
+    if (($setup_console =~ /x11/) && (check_var('RMT_TEST', 'rmt_chinese')) && (!is_sle('=15-SP5'))) {
```

Comment:
>=15-sp5

Reply:
rmt in maintenance (sle-15-sp4) honestly I don't remember, we could consider to remove the workaround, could you double-check?

Reply:
This case is running in Migration other job group while not Maintenance, anyway, I will double check.